webapp-django/crashstats/crashstats/admin.py

```diff
@@ -35,8 +36,7 @@ class LogEntryAdmin(admin.ModelAdmin):
     list_display = [
         'action_time',
         'user_email',
-        'content_type',
-        'object_repr',
+        'get_edited_object',
```

Comment:
This is a lot more useful than content_type + object_repr because the repr for users is useless.